spec/controllers/openid_connect/authorizations_controller_spec.rb

```diff
@@ -0,0 +1,172 @@
+require 'spec_helper'
+
+describe OpenidConnect::AuthorizationsController, type: :controller do
+  let!(:client) { OAuthApplication.create!(name: "Diaspora Test Client", redirect_uris: ["http://localhost:3000/"]) }
+  let!(:client_with_multiple_redirects) { OAuthApplication.create!(name: "Diaspora Test Client", redirect_uris: ["http://localhost:3000/","http://localhost/"]) }
```

Comment:
Might want to setup some factories in `spec/factories.rb`, see factory_girl's docs.